### PR TITLE
[WIP 🚧 ] Admin/Orders, behind feature toggle `admin_style_v2`: adjust colors

### LINKED ISSUE #10034: Change orders page colors
Work under `admin_style_v2`

Follow up on https://github.com/openfoodfoundation/openfoodnetwork/issues/9998

## Description
**- As a:** enterprise user
**- On page:** `/admin/orders`
**- I want to be able to do:**  Se BUU style on this page.

Which means: 

image

- New dropdowns in transparent background
- Table header in grey
- Perhaps use grey to continue distinguishing lines?
- Text in black (or the grey used in 9998 (which one is it?)
- For buttons (including edit/ship...) use : 

## Acceptance Criteria & Tests

1. Turn the feature toggl on
2. Check order page and see new colors
3. Check old colors are still appearing when feature toggl turned off

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

Closes #10034

#### Preview
screencapture-localhost-3000-admin-orders-2022-11-24-16_41_57

Capture d’écran 2022-11-24 à 16 49 49

Capture d’écran 2022-11-24 à 16 49 43

Capture d’écran 2022-11-24 à 16 49 37

#### What should we test?

- As an admin, with feature `admin_style_v2` activated, go to `/admin/order`: you should see the design
- As an admin, without feature `admin_style_v2`, nothing should change

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Basically, ignore everythings expect app/webpacker/controllers/*.js and app/webpacker/packs/*.js
 *.css
 *.scss
+# Except v2
+!/app/webpacker/css/admin/v2/**/*.scss
```
Comment (dacook) [inline .prettierignore:5]: I like the idea of using Prettier more, but once we merge v2 into the main code, we have to remove this line again, and choose:
* a) enable prettier on some or all of the css files, or
* b) not use prettier on css again.

Obviously (a) is the direction to head, but that makes it a pre-requisite for this task.
But it won't hurt to choose (b), at least this new code will be prettier!

**Review by dacook — APPROVED** [review]
I like how you've arranged the files and included header comments. I've reviewed the code, but will leave the visual testing to someone more familiar with this.

Thinking about this approach. This is using 'overrides', so they will have to be manually merged/overwritten later once ready. So far, it looks manageable.

**Comment by dacook** [comment]
Another idea is to set up a separate css package, which doesn't include these overridden files. 
That is, copy/paste `admin/all.scss` to `admin/all-v2.scss`, then one-by-one, switch over to the new v2 files.

Then it could be a bit simpler to switch over.

File: .prettierignore (hunk 1 above)
Comment (jibees) [reply .prettierignore:5]: I think solution `a)` is more satisfying: one solution for hat is probably to create a PR that run prettier on all scss files and then having well formatted files once and for all. 
What do you think?

**Comment by jibees** [comment]
> That is, copy/paste `admin/all.scss` to `admin/all-v2.scss`, then one-by-one, switch over to the new v2 files.

I must admit that I'm not sure to understand what you mean. 
Those v2 files include new css rules that override _previous_ rules that are less specific (due to the fact that we applies those files under a more specific selector which is `body.admin.admin-v2`). v2 files couldn't be used by itself: we should also include the files that they override.

**Comment by RachL** [comment]
@jibees some small changes and questions before moving to other pages: 

1. **When selecting an item in the drag and drop column, we don't see the result as it's white on white**: 

image

2 choices: we can apply the "small dropdown" design or the large one: 

image


2. **How about considering "clear filter" is a secondary button?** 

image

image

3. **It's super cool to have the action button in line but there is a strange effect on small size windows**: 

image

4. **I'm a bit lost on the color of the text within the table**: headers are ok but what kind of blue is used for links? I wonder if we shouldn't keep the same strong blue? As for the text in grey.... can we try a darker grey? What's used in back office UI uplift for content?

image


5. Let's use the same colors for links everywhere? Which means turning these in blue: 
image

**Comment by jibees** [comment]
1. Agree. Done.
2. I can have a try, but how about the next input type text field? They aren't very aligned

Capture d’écran 2022-11-28 à 11 17 56

2 bis. 
Yep, done
Capture d’écran 2022-11-28 à 11 18 38

3. Yes, thanks done.

4.
    - _what kind of blue is used for links?_ A lighter version of the new one (#0096ad) in order to have a hover effect (ie. then change to the new one #017a9a). But I tried a different version. Now we use the new one #017a9a as the default and a darker version when hovering. I let you test ;)
    - _As for the text in grey.... can we try a darker grey?_ Yes, we can try. I've changed to #666666
    -

5. Yep, we should. 
Capture d’écran 2022-11-28 à 11 35 16

**Comment by jibees** [comment]
Capture d’écran 2022-11-28 à 11 48 44

**Comment by jibees** [comment]
Capture d’écran 2022-11-28 à 11 54 45

**Comment by RachL** [comment]
As changes here are minimal since first review, I'm staging this again.

**Comment by RachL** [comment]
alright opening new issues for the next steps this one can be merged :+1:

**Pull request merged**